app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        print("Invalid command format.")
+        return
+
+    source_file = parts[1]
+    dest_file = parts[2]
+
+    if source_file == dest_file:
+        return
+
+    try:
+        with (open(source_file, "r") as file_in,
+              open(dest_file, "w") as file_out):
```

Comment:
Code Style: You have used the variable names 'file_in' and 'file_out' which are not descriptive enough. Consider renaming them to 'source_file' and 'destination_file' respectively.